sdks/python/apache_beam/io/parquetio_it_test.py

```diff
@@ -41,7 +41,14 @@
 from apache_beam.transforms import CombineGlobally
 from apache_beam.transforms.combiners import Count
 
+if not (platform.system() == 'Windows' and sys.version_info[0] == 2):
```

Comment:
Next time please use the more common format to handle missing modules used elsewhere in the codebase:
```py
try:
  import pyarrow as pa
except ImportError:
  pa = None
```

```py
@unittest.skipIf(pa is None, "...")
```